packages/iov-keycontrol/src/userprofile.ts

```diff
@@ -255,6 +256,32 @@ export class UserProfile {
     return this.primaryKeyring().createIdentity(walletId, chainId, options);
   }
 
+  /**
+   * Checks if an identitiy exists in the wallet with the given ID in the primary keyring
```

Comment:
"identity"

Reply:
The identitiy is with us for a very long time already. Kind of a tradition to spell it that way :P

https://github.com/iov-one/iov-core/pull/1202/commits/94bd268d42fddefcb1ef4547648820d5ae0c2150